### PR TITLE
Login UI 

### PR DESCRIPTION
**Pull request by Ananthavijay**
codepen link - https://codepen.io/Ananthavijay/pen/dQOrKo
Task link - https://codein.withgoogle.com/dashboard/task-instances/5705801445408768/

### CODE REVIEW

File: src/app/core/login/login.component.html [hunk 1]
```diff
@@ -1,3 +1,96 @@
...
+
+        </div>
+
+        
+      </div>
+   </div>
+   <div class="bottom">
+    <p>&copy JBoss Outreach 2018</p>
+  </div>
+  <script src='http://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js'></script>
```
Comment (nikitadhiman) [inline src/app/core/login/login.component.html]: no script src in html files, please include this in angular.json file under stylesheets. Thanks

**Comment by Ananthavijay** [comment]
done @nikitadhiman

**Comment by nikitadhiman** [comment]
Squash your commits, this is good to go then

**Comment by Ananthavijay** [comment]
done @nikitadhiman

**Pull request closed without merging**